server/models/user.js

```diff
@@ -28,7 +28,7 @@ userSchema.pre('save', function(next){
     })
 })
 
-userSchema.statics.comparedPassword = function(candidatePassword, cb) {
+userSchema.methods.comparedPassword = function(candidatePassword, cb) {
```

Comment:
The difference between statics and methods can be find here: http://stackoverflow.com/questions/29664499/mongoose-static-methods-vs-instance-methods

I think statics is more useful here because we want this model natively have this function as we go.(discussion opened!)